powerhome/playbook
# [PLAY-785] Converting Typeahead kit to Typescript

**What does this PR do?**
Converting Typeahead kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-785)

**How to test?**
1. Go to Typeahead kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
